src/main/java/gov/nasa/pds/tools/validate/rule/RegisterTargets.java

```diff
@@ -48,6 +49,9 @@ public void registerTargets() {
       try {
         Crawler crawler = getContext().getCrawler();
         WildcardOSFilter fileFilter = getContext().getFileFilters();
+        if (!"PDS4 Directory".equalsIgnoreCase(getContext().getRule().getCaption())) {
```

Comment:
This got removed in #734. Not sure why and did not leave myself a good comment. Without it, the filter is looking for just XML files.